e2e-tests/cypress/e2e/journeys/quote/your-quote/your-quote-multi-policy.spec.js

```diff
@@ -109,7 +109,7 @@ context('Get a quote/your quote page (multiple policy) - as an exporter, I want
 
         cy.checkText(row.key(), expectedKeyText);
 
-        const expected = '1.15%';
+        const expected = '1.13%';
```

Comment:
should these not come from constants?

Reply:
Do you have something in mind? There's a few quote tool tests that assert specific percentages and prices. This is already in `main` branch/production

Reply:
for later but we could have a const that does the calculation for us so we don't need to keep updating it manually

Reply:
That'd be good, we already have a calculation function (`generate-quote` directory). Without having a new shared package for helpers, constants etc, it could only be achieved by copying that directory and the pricing grid JSON into the E2E directory.